definitions/npm/xmldom_v0.x.x/test_xmldom_v0.x.x.js

```diff
@@ -0,0 +1,37 @@
+// @flow
```

Comment:
Please wrap the tests in a few [`describe/it`](https://github.com/flow-typed/flow-typed/blob/master/CONTRIBUTING.md#use-describe-and-it-blocks-to-limit-scope).